src/push.js

```diff
@@ -25,23 +25,23 @@ const TEMP_GCM_URL = 'https://gcm-http.googleapis.com/gcm';
 /**
  * URL safe Base64 encoder
  * @param  {Buffer} buffer The data to encode
- * @return {String}
+ * @return {String} URL safe base 64 encoded string
  */
 function ub64(buffer) {
   return buffer.toString('base64').replace(/\+/g, '-')
     .replace(/\//g, '_')
-    .replace(/=/g, '');
-};
+    .replace(/\=/g, '');
```

Comment:
Eslint throws an error for confusion between the referenced and:

var value \= 2;